types/index.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path="./locale/index.d.ts" />
```

Comment:
Not sure if we need this reference, can IDE get the path on its own, like our other plugins' types.

Reply:
No, There is no `locale/*.d.ts` files except `index` and `type`, but other plugins have their own `.d.ts`.